src/lib-rust/src/manager.rs

```diff
@@ -238,16 +238,16 @@ impl UpdateManager {
     }
 
     /// Get a list of available remote releases from the package source.
-    pub fn get_release_feed(&self) -> Result<VelopackAssetFeed, Error> {
+    pub fn get_release_feed(&self, staged_user_id: &str) -> Result<VelopackAssetFeed, Error> {
         let channel = self.get_practical_channel();
-        self.source.get_release_feed(&channel, &self.locator.get_manifest())
+        self.source.get_release_feed(&channel, &self.locator.get_manifest(), &staged_user_id)
     }
 
     /// Get a list of available remote releases from the package source.
     #[cfg(feature = "async")]
-    pub fn get_release_feed_async(&self) -> JoinHandle<Result<VelopackAssetFeed, Error>> {
+    pub fn get_release_feed_async(&self, staged_user_id: String) -> JoinHandle<Result<VelopackAssetFeed, Error>> {
```

Comment:
The public UpdateManager API probably shouldn't change here: the staged_user_id is known at this point in the UpdateManager locator